src/script/components/manifest-options.ts

```diff
@@ -0,0 +1,265 @@
+import { LitElement, customElement, css, html, property } from 'lit-element';
+
+@customElement('manifest-options')
+export class AppManifest extends LitElement {
+  @property({ type: Object }) manifest = {};
+
+  static get styles() {
+    return css``;
+  }
+
+  constructor() {
+    super();
+  }
+
+  render() {
+    return html`
+      <div>
+        <div class="head">
+          <!-- TODO borrow the manifest header from the report card -->
+          <div>
+            <h1>Manifest</h1>
+            <h1>Score 00/40</h1>
+          </div>
+
+          <h2>Summary</h2>
+          <div>
+            <p>
+              Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit
+              aut fugit, sed quia consequuntur magni dolores eos qui ratione
+              voluptatem sequi nesciunt. ven further!
+            </p>
+            <app-button @click=${this.done}>Done</app-button>
+          </div>
+        </div>
+        <fast-divider></fast-divider>
+        <section class="info">
+          <h1>Info</h1>
+          <div class="info-items inputs">${this.renderInfoItems()}</div>
+        </section>
+        <fast-divider></fast-divider>
+        <section class="images">
+          <h1>Images</h1>
+          <div class="icons">
+            <div class="images-header">
+              <h3>Upload App Icons</h3>
+              ${this.renderToolTip('TODO')}
+              <app-button @click=${this.openUploadModal}>Upload</app-button>
+            </div>
+            <div class="collection image-item">${this.renderIcons()}</div>
+
+            <div class="images-actions">
+              <app-button @click=${this.downloadImages}>Download</app-button>
+            </div>
+          </div>
+          <div class="screenshots">
+            <div class="screenshots-header">
+              <h3>Generate Screenshots</h3>
+              ${this.renderToolTip('TODO')}
+              <p>
+                Specify the URLs to generate desktop and mobile screenshots
+                from. You may add up to 8 screenshots or Store Listings.
+              </p>
+              <!-- url text field -->
+              <fast-text-field></fast-text-field>
```

Comment:
I know this is just mocked out at the moment, but lets make sure we are wrapping this section in a `form` element before we finalize this component.